active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,120 @@
+the Breadcrumb will 'crumble' starting at the root node. In addition,
+the leaf node can truncated if crumbling from the root node to the leaf node's parent is not enough for a resizing scenario. 
+
+A Breadcrumb control with 3 nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+A Breadcrumb "crumbling" with 3 visible nodes:
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
+
+A Breadcrumb control with the last node truncated:
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+The following Xaml markup  creates a simple Breadcrumbe control with three nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+ ~~~~ 
+     <muxc: Breadcrumb ItemsSource = {"Node", "Node", "Node"} /> 
+~~~~
+More examples to be added
+
+## Breadcrumb member notes
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | provides the content of the Breadcrumb | Null |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Spec note: the V2 of Breadcrumb will add flyouts from chevrons to view children of a node.
+Additional properties will be added to the API to enable this functionality. 
+
+# API Details
+
+```cs
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
+{
+    BreadcrumbLayout();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropdownItemTemplate{ get; set; };
+    Object ItemsSource{ get; set; };
+    Object ItemTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Breadcrumb, BreadcrumbItemClickedEventArgs> ItemClicked;
+
+    static Windows.UI.Xaml.DependencyProperty DropdownItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+}
+```
+
+# Appendix
+
+## UI Automation Pattern
+
+Breadcrumb will use the
+[InvokePattern](!https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-invoke-control-pattern).
+
+## Keyboard
+
+* The user can tab into the Breadcrumb and tab out.
+The first tab in gives focus to the first (highest-level) node.
+Once the Breadcrumb has focus, the user can use the left/right arrows to navigate from node to node.
+* If the Breadcrumb is crumbled, down/up will open the flyout from
+the ellipsis and down/up will navigate between the flyout items
+
+## Gamepad
+
+* Spatial navigation to focus and select the nodes.
+
+## Narrator
+
+When the focus is on a node, Narrator will announce the node
```

Comment:
How would the ellipsis button be read as? Should it state "Collapsed nodes" or some key phrase?

Reply:
Overflow, "More Breadcrumbs" - the ellipsis icon is used for **More** in the design language